src/resources/views/corporation/tracking.blade.php

```diff
@@ -3,83 +3,88 @@
-                @endif
-              </td>
-              <td>
-                <a href="{{ route('character.view.sheet', ['character_id' => $character->character_id]) }}">
-                  {!! img('character', $character->character_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
-                  <span class="id-to-name"
-                        data-id="{{$character->character_id}}">{{ trans('web::seat.unknown') }}</span>
-                </a>
-              </td>
-              <td>
-                <span class="character-id-to-main-character"
-                      data-character-id="{{$character->character_id}}">{{ trans('web::seat.unknown') }}</span>
-                @if(!is_null($character->ship_type_id))
-                  <i class="pull-right" data-toggle="tooltip" title="" data-original-title="{{ $character->type->typeName }}">
-                    {!! img('type', $character->ship_type_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
-                  </i>
-                @endif
-              </td>
-              <td>
-                @if(! is_null($character->location))
-                {{ $character->location->name }}
-                @else
-                Unknown Location
-                @endif
-              </td>
-              <td data-order="{{ $character->start_date }}">
-                <span data-toggle="tooltip" title="" data-original-title="{{ $character->start_date }}">
-                  {{ human_diff($character->start_date) }}
-                </span>
-              </td>
-              <td data-order="{{ $character->logon_date }}">
-                <span data-toggle="tooltip" title="" data-original-title="{{ $character->logon_date }}">
-                  {{ human_diff($character->logon_date) }}
-                </span>
-              </td>
-            </tr>
-
-        @endforeach
-
-        </tbody>
       </table>
 
     </div>
-    <div class="panel-footer">Registered users <b>{{ $tracking->where('key_ok', true)->count() }} / {{ $tracking->count() }}</b></div>
   </div>
 
 @stop
 
 @push('javascript')
-  @include('web::includes.javascript.character-id-to-main-character')
+
+  <script>
+
+    function refreshTokenCheckboxes() {
+      return $('#refresh_token_status > input[type="checkbox"]').map(function(){
+        if(this.checked)
+          return $(this).val()
+      }).get().join();
+    }
+
+    $( '#refresh_token_status > input[type="checkbox"]' ).click(function() {
+      corporation_member_tracking_table.draw();
+    });
+
+    var corporation_member_tracking_table = $('table#corporation-member-tracking').DataTable({
+      processing      : true,
+      serverSide      : true,
+      ajax            : {
+        url : '{{ route('corporation.view.tracking.data', ['corporation_id' => $request->corporation_id]) }}',
+        data: function ( d ) {
+          d.selected_refresh_token_status = refreshTokenCheckboxes();
```

Comment:
why not sending the form itself with both field and their attached values ?
It will reduce complexity :)